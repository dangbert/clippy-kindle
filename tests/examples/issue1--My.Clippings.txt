How to Own the World: A Plain English Guide to Thinking Globally and Investing Wisely: The new edition of the life-changing personal finance bestseller (Craig, Andrew)
- Your Highlight on page 24 | location 314-315 | Added on Wednesday, 14 October 2020 22:27:14

you need to act on it.
==========
How to Own the World: A Plain English Guide to Thinking Globally and Investing Wisely: The new edition of the life-changing personal finance bestseller (Craig, Andrew)
- Your Highlight on page 28 | location 373-374 | Added on Wednesday, 14 October 2020 23:02:31

‘It is not because things are difficult that we dare not venture. It is because we dare not venture that they are difficult.’
==========
How to Own the World: A Plain English Guide to Thinking Globally and Investing Wisely: The new edition of the life-changing personal finance bestseller (Craig, Andrew)
- Your Highlight on page 38 | location 516-517 | Added on Thursday, 15 October 2020 07:23:50

no company should ever be worth 25 times the value of its sales or a few hundred
==========
How to Own the World: A Plain English Guide to Thinking Globally and Investing Wisely: The new edition of the life-changing personal finance bestseller (Craig, Andrew)
- Your Highlight on page 38 | location 516-517 | Added on Thursday, 15 October 2020 07:24:06

no company should ever be worth 25 times the value of its sales or a few hundred times the value of its profits.
==========
How to Own the World: A Plain English Guide to Thinking Globally and Investing Wisely: The new edition of the life-changing personal finance bestseller (Craig, Andrew)
- Your Highlight on page 51 | location 690-690 | Added on Thursday, 15 October 2020 14:00:25

There is safety in the herd.
==========
Ikigai: The Japanese Secret to a Long and Happy Life (Héctor García;Francesc Miralles)
- Your Highlight at location 177-178 | Added on Saturday, 21 November 2020 00:35:37

If a member of a moai is in financial trouble, he or she can get an advance from the group’s savings.
==========
Ikigai: The Japanese Secret to a Long and Happy Life (Héctor García;Francesc Miralles)
- Your Note at location 178 | Added on Saturday, 21 November 2020 00:36:11

Japanese Pardna
==========
How to Own the World: A Plain English Guide to Thinking Globally and Investing Wisely: The new edition of the life-changing personal finance bestseller (Craig, Andrew)
- Your Bookmark on page 262 | location 3686 | Added on Friday, 23 October 2020 21:24:06


==========
