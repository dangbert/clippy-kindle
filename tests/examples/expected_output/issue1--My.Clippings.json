[
  {
    "title": "\ufeffHow to Own the World: A Plain English Guide to Thinking Globally and Investing Wisely: The new edition of the life-changing personal finance bestseller",
    "author": "Craig, Andrew",
    "dateStart": "October 14, 2020 22:27:14",
    "dateEnd": "October 14, 2020 22:27:14",
    "items": [
      {
        "type": "highlight",
        "loc": 314,
        "locEnd": 315,
        "locType": "location",
        "dateStr": "October 14, 2020 22:27:14",
        "content": "you need to act on it."
      }
    ]
  },
  {
    "title": "How to Own the World: A Plain English Guide to Thinking Globally and Investing Wisely: The new edition of the life-changing personal finance bestseller",
    "author": "Craig, Andrew",
    "dateStart": "October 14, 2020 23:02:31",
    "dateEnd": "October 23, 2020 21:24:06",
    "items": [
      {
        "type": "highlight",
        "loc": 373,
        "locEnd": 374,
        "locType": "location",
        "dateStr": "October 14, 2020 23:02:31",
        "content": "\u2018It is not because things are difficult that we dare not venture. It is because we dare not venture that they are difficult.\u2019"
      },
      {
        "type": "highlight",
        "loc": 516,
        "locEnd": 517,
        "locType": "location",
        "dateStr": "October 15, 2020 07:23:50",
        "content": "no company should ever be worth 25 times the value of its sales or a few hundred"
      },
      {
        "type": "highlight",
        "loc": 516,
        "locEnd": 517,
        "locType": "location",
        "dateStr": "October 15, 2020 07:24:06",
        "content": "no company should ever be worth 25 times the value of its sales or a few hundred times the value of its profits."
      },
      {
        "type": "highlight",
        "loc": 690,
        "locEnd": 690,
        "locType": "location",
        "dateStr": "October 15, 2020 14:00:25",
        "content": "There is safety in the herd."
      },
      {
        "type": "bookmark",
        "loc": 3686,
        "locType": "location",
        "dateStr": "October 23, 2020 21:24:06"
      }
    ]
  },
  {
    "title": "Ikigai: The Japanese Secret to a Long and Happy Life",
    "author": "H\u00e9ctor Garc\u00eda;Francesc Miralles",
    "dateStart": "November 21, 2020 00:35:37",
    "dateEnd": "November 21, 2020 00:36:11",
    "items": [
      {
        "type": "highlight",
        "loc": 177,
        "locEnd": 178,
        "locType": "location",
        "dateStr": "November 21, 2020 00:35:37",
        "content": "If a member of a moai is in financial trouble, he or she can get an advance from the group\u2019s savings."
      },
      {
        "type": "note",
        "loc": 178,
        "locType": "location",
        "dateStr": "November 21, 2020 00:36:11",
        "content": "Japanese Pardna"
      }
    ]
  },
  {
    "title": "The Slight Edge: Turning Simple Disciplines into Massive Success and Happiness",
    "author": "Jeff Olson",
    "dateStart": "December 07, 2020 19:19:23",
    "dateEnd": "December 07, 2020 19:19:23",
    "items": [
      {
        "type": "note",
        "loc": 231,
        "locType": "location",
        "dateStr": "December 07, 2020 19:19:23",
        "content": "Twebty two"
      }
    ]
  }
]